.38 special and revolvers

FluffiestFloof opened the pull request:
<!-- The text between the arrows are comments - they will not be visible on your PR. -->
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR <!-- Describe the Pull Request here. What does it change? What other things could this impact? -->
Adds .38 ammo alongside four revolvers that uses that specific ammo. Damage is slightly lower than magnum ammo from the damage tweak thread.
Primarily PR'd for the whole HoP Snub Revolver but figured that only one revolver with it's own ammo was a bit eh so made more than one.

- Adds "Lucky 37" a 6 rounds .38 revolver meant to really just have another revolver that uses the .38 revolver ammo. 
- Adds "K-38 Masterpiece" a 6 rounds .38 revolver meant for Security usage.
- Adds "Fitz Special" a 6 rounds .38 revolver meant for the HoP that comes pre-loaded with rubber bullets.
- Adds "Faith" a 4 rounds .38 revolver meant for some holy person pre-loaded with holy bullets.
- Adds the .38 ammo (normal, practice, rubber, high-velocity, holy), ammo box (except holy) and speedloaders (except holy).

None of the guns are currently set to spawn.
The ammo recipe is in the lathe.
The damage for the projectiles are roughly based on the damage rework.

**Media**
![ss+(2023-03-15+at+01 15 07)](https://user-images.githubusercontent.com/9823203/225170477-621adf39-0c96-4b0c-bbb6-8300c493dd4f.png)
Lucky 37, K-38 Masterpiece, Fitz Special, Faith

![ss (2023-03-02 at 01 43 47)](https://user-images.githubusercontent.com/9823203/225167303-1907333d-c91b-4e46-8a8f-95304b549873.png)
Ammo types (Normal, HighVelocity, Rubber and Practice)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
-->

:cl:
- add: Added three revolvers using .38 special.

FluffiestFloof committed:
.38 pew pew

FluffiestFloof committed:
Fix

How did I do this?

FluffiestFloof committed:
Police Special

FluffiestFloof committed:
HoP gets the pew

FluffiestFloof committed:
Modify Faith

Give it a even smaller feel.

FluffiestFloof committed:
Revert Golemancer having special bible

FluffiestFloof committed:
snubs the hop revolver slightly more

Elijahrane commented on Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Chapel/bibles.yml:
```diff
@@ -13,3 +13,22 @@
+    capacity: 10
+  - type: StorageFill
+    contents:
+      - id: WeaponRevolverFaith
```
when originally designing the chaplain for ss14 there was a discussion over what to give them (null rod, faith, something else?) and we decided on familiars filling this role

OldDanceJacket replied:
Yes, but having it in the game would still be nice, especially given some of the potential ideas for the future of Nyano.

Elijahrane replied:
why does this role need a gun...?

OldDanceJacket replied:
It's not the role it's the gun itself being in the game. Lmao.

Vordenburg replied:
For the gun to be in the game, it has to come from somewhere. There's no need to have an unused entity.

FluffiestFloof replied:
I would argue that ~~80%~~ half of the guns in the game have to be mapped in anyway but I'll remove Faith and the special bible (that I originally forgot to remove anyway) since a majority doesn't see a need for it

FluffiestFloof committed:
Remove Faith and related items

Majority won

FluffiestFloof committed:
Remade sprites for the upstream gun update

Also made an actual tiny cute gun for hop

FluffiestFloof committed:
Fixes

Make meta more accurate for snub, forgot to move the rubber bullets to snub and modify the snub sprite even more.

FluffiestFloof committed:
Updated masterpiece sprite

It felt too flat compared to the gradient of the Lucky or Snub.

FluffiestFloof committed:
Removed guns from spawns and readd Faith

Lucky removed from Salvage, Fitz removed from HoP locker and readds holy rounds alongside Faith.
Also changed sprites again since I'm unable to decide...

FluffiestFloof committed:
Merge branch 'Nyanotrasen:master' into snubNose

FluffiestFloof committed:
Merge branch 'master' into snubNose

FluffiestFloof committed:
tags.yml

Just to separate it

FluffiestFloof committed:
Merge branch 'master' into snubNose
